salt/modules/boto_route53.py

```diff
@@ -254,29 +254,29 @@ def zone_exists(
     """
     Check for the existence of a Route53 hosted zone.
 
-    .. versionadded:: 2015.8.0
+     .. versionadded:: 2015.8.0
```

Comment:
had this on the original review but should have specified for the entire doc for this method everything is shifted to the right. Shouldn't it align with the `"""` above?